vi/dot_file.py

```diff
@@ -31,20 +31,20 @@ def run(self):
     def parse(self, line):
         try:
             _logger().info('[DotFile] parsing line: {0}'.format(line))
-            if line.startswith((':map ')):
-                line = line[1:]
+            line = line.lstrip(':')

```

Comment:
Empirically vim seems to ignore trailing and leading whitespace in .vimrc lines, so the code could probably be simplified with a line = line.trim() first thing, then you can remove the rstrip.